CIP-????/README.md

```diff
@@ -0,0 +1,294 @@
+---
+CIP: ????
+title: PostQuantum signatures and native wallets
+Category: Plutus
+author: Michał J. Gajda (mjgajda@migamake.com)
+Implementors: []
+Discussions-To: https://github.com/cardano-foundation/CIPs/pull/441
+Comments-Summary: 
+Comments-URI:  
+Status: Proposed
+Type: Standards Track
+Created: 2023-01-19  
+License: Apache-2.0
+---
+## Simple Summary
+
+Support post-quantum signatures (Dilithium+CRYSTALS, FALCON) as recommended by NIST standardization effort in Plutus Core;
+allow validation of such signatures as builtins.
+NIST plans to finalize standardization in 2023/2024 time frame, with US executive branch recommending
+all projects to plan for post-quantum resistance by mid-2023.
+We should prepare Cardano Blockchain for migration.
+
+## Abstract
+
+Provides a way of verifying post-quantum signatures as recommended by NIST in Plutus Core.
+New builtins will allow use of these signatures within smart contracts. These builtins work over
+`BuiltinByteString`s.
+
+## Motivation
+
+NIST estimates that pre-quantum cryptographic algorithms may be no longer safe
+by 2035. However, the smart contracts have long development and lifetime,
+so migration to post-quantum cryptography algorithms should start several
+years before they would become insecure.
+This summer NIST has recommended three PQC signature algorithms
+for standardization, publishing their reference implementations,
+and finishing initial comment period.
+
+Supporting PQC signatures will bring the following benefits:
+
+* smart contracts may use future standard algorithms today;
+* early support for PQC will simplify future PQC migration, if needed;
```

Comment:
it's unclear to me what the PCQ migration is and how this enables it? I would like to see this elaborated somewhere. It seems that you have an idea of what it is and how we would do it, but it's not explained anywhere.